app/Http/Controllers/GroupController.php

```diff
@@ -23,6 +25,34 @@ public function __construct()
         $this->beforeFilter('auth', array('on' => array('post', 'put', 'delete')));
     }
 
+    /**
+     * Store newly created Group.
+     *
+     * @return Response
+     */
+    public function store(StoreGroupRequest $request)
```

Comment:
As for `create` vs `store`, I went with `store` to match [Laravel conventions](https://laravel.com/docs/5.1/controllers#restful-resource-controllers) here.